attendence-app0.postman_collection.json

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
Don't post this file on your project root. It should not be here .If you want to share this, add a link to it in the readme

Reply:
It’s postman export. Since Api are getting changed frequently so thought to put this for sometime as it’s easy to export and push here rather than generate docs and edit readme.
It can be simply imported and checked by someone